lib/api/v3/queries/schemas/filter_dependency_representer_factory.rb

```diff
@@ -52,21 +52,25 @@ def create(filter, operator, form_embedded: false)
           private
 
           @specific_conversion = {
-            'CreatedAtFilter': 'DateTimeFilter',
-            'UpdatedAtFilter': 'DateTimeFilter',
-            'AuthorFilter': 'UserFilter',
-            'ResponsibleFilter': 'AllPrincipalsFilter',
-            'AssignedToFilter': 'AllPrincipalsFilter',
-            'WatcherFilter': 'UserFilter'
+            CreatedAtFilter: 'DateTimeFilter',
+            UpdatedAtFilter: 'DateTimeFilter',
+            AuthorFilter: 'UserFilter',
+            ResponsibleFilter: 'AllPrincipalsFilter',
+            AssignedToFilter: 'AllPrincipalsFilter',
+            WatcherFilter: 'UserFilter'
           }
 
+          @excluded_filters = [
+            ::Queries::WorkPackages::Filter::RelatableFilter
+          ]
```

Comment:
If you want to exclude filters from within a module e.g. storage we will need to add a method for doing so. Don't know if you want to do this within the scope of this PR, @Kharonus.

Reply:
How is it different, when the filter is defined in a module?

Reply:
In order to keep the separation I'd expect some sort of registration mechanism in the module engine, e.g. 

```
config.to_prepare do
 API::...::FilterDependencyRepresenterFactory.add_excluded_filters ::MyFilter
end
```

Reply:
Ah, so you wouldn't add it directly into the factory but have a register hook. Agreed.